tests/acceptance/28_inform_testing/01_files/changes.cf.expected

```diff
@@ -0,0 +1,14 @@
+    info: Edit file '/tmp/TEST.cfengine'
+    info: files promise '/tmp/TEST.cfengine' repaired
```

Comment:
hmm, looks like maybe we should add *file:LINE_NUMBER* to the log message, maybe?

Reply:
That would probably be a good idea. :+1: